### PR TITLE
Fix typos in outputs/docs/src/etc.

### PR DESCRIPTION
**Pull request by shorowit**
Pull request overview
---------------------
 - Fixes various typos in docs, src/test, IDD.
 - IDD changes are all in notes, so not a breaking change.
 - A couple src changes (that I point out below) change outputs

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EnergyPlus/ChillerAbsorption.cc [hunk 1]
```diff
@@ -1222,7 +1222,7 @@ void BLASTAbsorberSpecs::sizeChiller(EnergyPlusData &state)
                             BaseSizer::reportSizerOutput(state,
                                                          moduleObjectType,
                                                          this->Name,
-                                                         "Iniital Design Size Design Generator Fluid Flow Rate [m3/s]",
+                                                         "Initial Design Size Design Generator Fluid Flow Rate [m3/s]",
```
Comment (shorowit) [inline src/EnergyPlus/ChillerAbsorption.cc:1225]: Seems like this affects output?

File: src/EnergyPlus/DXCoils.cc [hunk 2]
```diff
@@ -14702,7 +14702,7 @@ void CalcTwoSpeedDXCoilStandardRating(EnergyPlusData &state, int const DXCoilNum
             newPreDefSubTable(state, state.dataOutRptPredefined->pdrEquip, "VAV DX Cooling Standard Rating Details");
         state.dataOutRptPredefined->pdchVAVDXCoolCoilType =
             newPreDefColumn(state, state.dataOutRptPredefined->pdstVAVDXCoolCoil, "DX Cooling Coil Type");
-        state.dataOutRptPredefined->pdchVAVDXFanName = newPreDefColumn(state, state.dataOutRptPredefined->pdstVAVDXCoolCoil, "Assocated Fan");
+        state.dataOutRptPredefined->pdchVAVDXFanName = newPreDefColumn(state, state.dataOutRptPredefined->pdstVAVDXCoolCoil, "Associated Fan");
```
Comment (shorowit) [inline src/EnergyPlus/DXCoils.cc:14705]: Affects output?

File: src/EnergyPlus/HeatRecovery.cc [hunk 3]
```diff
@@ -1666,7 +1666,7 @@ namespace HeatRecovery {
                                                  this->Name,
                                                  this->type == HVAC::HXType::AirToAir_FlatPlate
                                                      ? "Flat Plate"
-                                                     : (this->type == HVAC::HXType::Desiccant_Balanced ? "Dessicant Balanced" : "Generic"));
+                                                     : (this->type == HVAC::HXType::Desiccant_Balanced ? "Desiccant Balanced" : "Generic"));
```
Comment (shorowit) [inline src/EnergyPlus/HeatRecovery.cc:1669]: Affects output?

File: src/EnergyPlus/HybridUnitaryAirConditioners.cc [hunk 4]
```diff
@@ -1117,43 +1117,43 @@ void GetInputZoneHybridUnitaryAirConditioners(EnergyPlusData &state, bool &Error
...
                             "Zone Hybrid Unitary HVAC Dehumidification Load to Humidistat Setpoint Heat Transfer Rate",
                             Constant::Units::W,
-                            hybridUnitaryAC.RequestedDeHumdificationLoad,
+                            hybridUnitaryAC.RequestedDeHumidificationLoad,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Average,
                             hybridUnitaryAC.Name);
         SetupOutputVariable(state,
-                            "Zone Hybrid Unitary HVAC DehumidificationLoad to Humidistat Setpoint Heat Tansfer Energy",
+                            "Zone Hybrid Unitary HVAC Dehumidification Load to Humidistat Setpoint Heat Transfer Energy",
```
Comment (shorowit) [inline src/EnergyPlus/HybridUnitaryAirConditioners.cc:1132]: Affects output?

File: src/EnergyPlus/HybridUnitaryAirConditioners.cc [hunk 5]
```diff
@@ -1117,43 +1117,43 @@ void GetInputZoneHybridUnitaryAirConditioners(EnergyPlusData &state, bool &Error
...
                             "Zone Hybrid Unitary HVAC Humidification Load to Humidistat Setpoint Heat Transfer Rate",
                             Constant::Units::W,
-                            hybridUnitaryAC.RequestedHumdificationLoad,
+                            hybridUnitaryAC.RequestedHumidificationLoad,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Average,
                             hybridUnitaryAC.Name);
         SetupOutputVariable(state,
-                            "Zone Hybrid Unitary HVAC Humidification Load to Humidistat Setpoint Heat Tansfer Energy",
+                            "Zone Hybrid Unitary HVAC Humidification Load to Humidistat Setpoint Heat Transfer Energy",
```
Comment (shorowit) [inline src/EnergyPlus/HybridUnitaryAirConditioners.cc:1154]: Affects output?

Comment (shorowit) [reply src/EnergyPlus/HybridUnitaryAirConditioners.cc:1154]: A bunch of these HybridUnitaryAC outputs don't appear in the documentation.

File: src/EnergyPlus/SteamBaseboardRadiator.cc [hunk 6]
```diff
@@ -1232,7 +1232,7 @@ namespace SteamBaseboardRadiator {
                                                          state.dataSteamBaseboardRadiator->SteamBaseboard(BaseboardNum).EquipID,
                                                          "Design Size Maximum Steam Flow Rate [m3/s]",
                                                          SteamVolFlowRateMaxDes,
-                                                         "User-Speicified Maximum Steam Flow Rate [m3/s]",
+                                                         "User-Specified Maximum Steam Flow Rate [m3/s]",
```
Comment (shorowit) [inline src/EnergyPlus/SteamBaseboardRadiator.cc:1235]: Affects output?

File: src/EnergyPlus/SurfaceGeometry.cc [hunk 7]
```diff
@@ -7484,7 +7484,7 @@ namespace SurfaceGeometry {
         // Write reflectance values to .eio file.
         print(state.files.eio,
               "! <ShadingProperty Reflectance>,Shading Surface Name,Shading Type,Diffuse Solar Reflectance, Diffuse "
-              "Visible Reflectance,Surface Glazing Fraction,Surface Glazing Contruction\n");
+              "Visible Reflectance,Surface Glazing Fraction,Surface Glazing Construction\n");
```
Comment (shorowit) [inline src/EnergyPlus/SurfaceGeometry.cc:7487]: Affects output?

**Comment by shorowit** [comment]
I looked through a bunch of the regression files and they are all typo fixes to column names, etc. This should be good to go, other than needing to mention all these output changes in OutputChanges.md (which I could use some help with).

File: src/EnergyPlus/ChillerAbsorption.cc (hunk 1 above)
Comment (Myoldmopar) [reply src/EnergyPlus/ChillerAbsorption.cc:1225]: Yeah probably worth mentioning at least.  I can try to collect the changes here that "affect" output and add it to the new 25.1 output changes file.

File: src/EnergyPlus/DXCoils.cc (hunk 2 above)
Comment (mjwitte) [reply src/EnergyPlus/DXCoils.cc:14705]: > Affects output?

Yes

File: src/EnergyPlus/HeatRecovery.cc (hunk 3 above)
Comment (mjwitte) [reply src/EnergyPlus/HeatRecovery.cc:1669]: > Affects output?

Yes

File: src/EnergyPlus/HybridUnitaryAirConditioners.cc (hunk 4 above)
Comment (mjwitte) [reply src/EnergyPlus/HybridUnitaryAirConditioners.cc:1132]: > Affects output?

Yes

File: src/EnergyPlus/HybridUnitaryAirConditioners.cc (hunk 5 above)
Comment (mjwitte) [reply src/EnergyPlus/HybridUnitaryAirConditioners.cc:1154]: > Affects output?

Yes

File: src/EnergyPlus/SteamBaseboardRadiator.cc (hunk 6 above)
Comment (mjwitte) [reply src/EnergyPlus/SteamBaseboardRadiator.cc:1235]: > Affects output?

Yes

File: src/EnergyPlus/SurfaceGeometry.cc (hunk 7 above)
Comment (mjwitte) [reply src/EnergyPlus/SurfaceGeometry.cc:7487]: > Affects output?

Yes

File: src/EnergyPlus/ThermalComfort.cc [hunk 8]
```diff
@@ -169,7 +169,7 @@ namespace ThermalComfort {
             CurrentGroupName = state.dataHeatBal->People(Loop).Name;
 
             // CurrentModuleObject='People'
-            // MJW MRT ToDo: Rename most Zone Therml Comfort output varibles to People Thermal Comfort ('cause they're keyed by People name)
+            // MJW MRT ToDo: Rename most Zone Thermal Comfort output variables to People Thermal Comfort ('cause they're keyed by People name)
```
Comment (mjwitte) [inline src/EnergyPlus/ThermalComfort.cc:172]: Hmmm, "MJW" should follow up on this.

**Review by mjwitte — APPROVED** [review]
@Myoldmopar @shorowit Sooo many changes, surprising, but no surprising.

I've added the output changes and output variables csv changes. Please double-check.

**Comment by shorowit** [comment]
> @Myoldmopar @shorowit Sooo many changes, surprising, but no surprising.
> 
> I've added the output changes and output variables csv changes. Please double-check.

@mjwitte Yeah, I started off fixing one typo in HTML output and then went down a rabbit hole...

Your changes to OutputChanges24-2-0-to-25-1-0.md look reasonable to me, thanks for doing that. Should the ChillerAbsorption change ("Iniital" -> "Initial") be documented somewhere too? I can't immediately tell where that is output and I don't see it in any CI diffs.

**Comment by mjwitte** [comment]
>  Your changes to OutputChanges24-2-0-to-25-1-0.md look reasonable to me, thanks for doing that. Should the ChillerAbsorption change ("Iniital" -> "Initial") be documented somewhere too? I can't immediately tell where that is output and I don't see it in any CI diffs.

Yes, I missed that one. If I'm following the code correctly, you should see this if you turn on "Do HVAC Sizing Simulation" for a file that has a Chiller:Absorption.

**Comment by mjwitte** [comment]
@shorowit Took a few tries, but managed to produce the diff for the absorption chiller using 
[AbsorptionChiller-Autosized.idf.txt](https://github.com/user-attachments/files/17513455/AbsorptionChiller-Autosized.idf.txt)

image

The initial sizes are reported twice for some reason, but not important here.

Finished with this PR. @Myoldmopar ready for final review and merge.

**Comment by Myoldmopar** [comment]
Everything is passing here, let's get this in.  Thanks @shorowit

**Pull request merged**